rpcs3/util/serialization.hpp

```diff
@@ -0,0 +1,252 @@
+#pragma once
+
+#include "util/types.hpp"
+#include <vector>
+
+struct serial_load
```

Comment:
Can't you call it serialize and deserialize?

Reply:
Or serializer and deserializer

Reply:
What about serial and deserial? I just can predict the many typos people will make with full naming.

Reply:
ok changed.